reference/kubectl-plugin.md

```diff
@@ -188,4 +187,8 @@ The plugin requires access to the `Mayastor REST server` for execution. It gets
 ### Limitations:
 
 - The plugin currently does not have authentication support.
-- The plugin can operate only over HTTP.
\ No newline at end of file
+- The plugin can operate only over HTTP.
+
+
+
```

Comment:
Too many extra lines here, are these expected?

Reply:
Fixed it.